urbanairship-react-native/tsconfig.json

```diff
@@ -8,6 +8,6 @@
     "target": "es5",
     "noImplicitAny": true
   },
-  "include": ["js/**/*.ts"],
+  "include": ["js/**/*.ts", "js/**/*.tsx"],
```

Comment:
I added the `tsx` files to pick up MessageView. One concern I have is that MessageView will now be added to the build, and maybe we didn't want it there?

Reply:
@marc-scig I assume we want it part of the build?